lib/github-client.js

```diff
@@ -192,4 +222,20 @@ GithubClient.prototype.updatePR = function (pr, data) {
   })
 }
 
+GithubClient.prototype.addReleasePRLabel = function (prNo) {
```

Comment:
Is it necessary to add the label with separate API call? 
Can you set the label when creating the PR in `prepareReleasePR`?

This way script runs faster with less API calls.

Reply:
Okay, thanks for checking! Approved.